examples/beam/src/test/java/edu/snu/nemo/examples/beam/PartitionWordsByLengthITCase.java

```diff
@@ -65,8 +67,36 @@ public void tearDown() throws Exception {
   @Test (timeout = TIMEOUT)
   public void test() throws Exception {
     JobLauncher.main(builder
-      .addJobId(PartitionWordsByLength.class.getSimpleName())
-      .addOptimizationPolicy(DefaultPolicyParallelismFive.class.getCanonicalName())
-      .build());
+        .addResourceJson(executorResourceFileName)
+        .addJobId(PartitionWordsByLengthITCase.class.getSimpleName())
+        .addOptimizationPolicy(DefaultPolicyParallelismFive.class.getCanonicalName())
+        .build());
+  }
+
+  @Test (timeout = TIMEOUT)
+  public void testSailfish() throws Exception {
```

Comment:
It seems that there are too many IT cases for this example. In my opinion, it would be enough to have two tests which test default policy and one optimization policy (sailfish or pado).

Reply:
I agree. I will leave only the default and sailfish policy.